controller_manager:
  ros__parameters:
    update_rate: 1000  # Hz

    # Define the controllers and their types here
    arm_controller:
      type: effort_controllers/JointGroupEffortController

    joint_state_broadcaster:
      type: joint_state_broadcaster/JointStateBroadcaster

# Configuration for the Arm Controller
arm_controller:
  ros__parameters:
    joints:
      - joint1
    command_interfaces:
      - effort
    state_interfaces:
      - position
      - velocity

# Configuration for the Joint State Broadcaster
joint_state_broadcaster:
  ros__parameters:
    # Watching the single pendulum joint
    joints:
      - joint1
    interfaces:
      - position
      - velocity
      - effort